contracts/account/utils/ERC6900Utils.sol

```diff
@@ -0,0 +1,124 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.20;
+
+import {ValidationConfig, ModuleEntity, ValidationFlags, HookConfig, ExecutionManifest, ManifestExecutionHook, ManifestExecutionFunction, Call} from "../../interfaces/IERC6900.sol";
+import {Bytes} from "@openzeppelin/contracts/utils/Bytes.sol";
+import {Packing} from "@openzeppelin/contracts/utils/Packing.sol";
+import {Address} from "@openzeppelin/contracts/utils/Address.sol";
+
+/**
+ * @dev Library with common ERC-6900 utility functions.
+ *
+ * See https://eips.ethereum.org/EIPS/eip-6900[ERC-6900].
+ */
+// slither-disable-next-line unused-state
+library ERC6900Utils {
+    using Packing for *;
+
+    error ERC6900ModuleInterfaceNotSupported(address module, bytes4 expectedInterface);
+    error ERC6900AlreadySetSelectorForValidation(ModuleEntity validationFunction, bytes4 selector);
+    error ERC6900AlreadySetValidationHookForValidation();
+    error ERC6900AlreadySetExecutionHookForExecution();
+    error ERC6900AlreadySetExecutionHookForValidation();
+    error ERC6900AlreadyUsedModuleFunctionExecutionSelector(bytes4 selector);
+    error ERC6900ExecutionSelectorConflictingWithERC4337Function(address module, bytes4 selector);
+    error ERC6900ExecutionSelectorConflictingWithERC6900Function(address module, bytes4 selector);
+    error ERC6900BadUserOpValidation(ModuleEntity moduleEntity);
+    error ERC6900BadSignatureValidation(ModuleEntity moduleEntity);
+    error ERC6900MissingValidationForSelector(bytes4 selector);
+    error ERC6900ExecutionSelectorNotAllowedForGlobalValidation(
+        ModuleEntity validationModuleEntity,
+        bytes4 executionSelector
+    );
+    error ERC6900InvalidExecuteTarget();
+    error ERC6900ExecutionFailed(address target, uint256 value, bytes data);
```

Comment:
If the error are not emited in the utils library, we usually define them in the contract itself

Reply:
Ok, see [Move errors to AccountERC6900](https://github.com/OpenZeppelin/openzeppelin-community-contracts/pull/80/commits/dbb5521286bcd9ff10343c93be7109a6940674a3).